snippet.js

```diff
@@ -90,6 +90,15 @@
         }
     }
 
+    // set version query parameters
+    var versions = window.mParticle.config.versions;
+    if (versions) {
+        var versionQueryArray = [];
```

Comment:
`versionQueryArray` should be moved outside of this if block since it's being referenced on line 115.